scripts/test.sh

```diff
@@ -99,3 +99,12 @@ if [[ $TEST_SUITE == "unit" ]]; then
 	#     done
 	# fi
 fi
+
+# elif [[ $TEST_SUITE == "integration" ]]; then
+# 	cd $SNAP_PLUGIN_SOURCE
+# 	if [[ $SNAP_PLUGIN_SOURCE == "" ]]; then
+# 	echo "env var SNAP_PLUGIN_SOURCE not set, should be location of snap-plugin-publisher-hana"
+# 	exit 1
+# 	fi
+# 	go test -v --tags=integration ./...
+# fi
```

Comment:
I guess leave it in for now since we are planning to refactor tests to address this exact problem.